src/OrchardCore/OrchardCore.Setup.Core/SetupService.cs

```diff
@@ -202,7 +202,8 @@ await scope
                     context.AdminPassword,
                     context.DatabaseProvider,
                     context.DatabaseConnectionString,
-                    context.DatabaseTablePrefix
+                    context.DatabaseTablePrefix,
+                    Properties = context.Properties
```

Comment:
Can you change the `ExecuteAsync` call to use a `Dictionary<string, object>` instead of an anonymous object like today, and merge the `Properties` values inside this new dictionary?

It won't change how `parameters()` is called since the object is transformed in `JObject`, but your properties will also be available directly like `parameters('FirstName')`.

Reply:
done. Please review.